recipe-box-frontend/src/main/frontend/src/components/Header/Header.tsx

```diff
@@ -23,23 +23,16 @@ const Header: React.FC = () => {
                                 <li >
                                 {userType === UserType.CHEF && (
                                     <>
-                                        <button 
-                                        className="nav-item" 
-                                        onClick={() => setIsOpen(!isOpen)}
-                                        >
-                                        Recipes
-                                        <ChevronDown className='chevron-item'></ChevronDown>
-                                        </button>
-                                        
-                                        
-                                        {isOpen && (
-                                        <ul className="dropdown-menu" onClick={() => setIsOpen(!isOpen)}>
-                                            <li><Link to="/recipes/list" className="dropdown-item">Public Recipes</Link></li>
-                                            <li><Link to="/recipes/myRecipes" className="dropdown-item">My Recipes</Link></li>
-                                            <li><Link to="/recipes/ingredients" className="dropdown-item">Recipes with My Ingredients</Link></li>
-                                        </ul>
-                                        )}
-                                    </>
+                                    <li className='left'>
+                                    <Dropdown buttonText="Recipes" content={
+                                        <>
+                                            <Link to="/recipes/list" className="dropdown-item">Public recipes</Link>
+                                            <Link to="/recipes/myRecipes" className="dropdown-item">My recipes</Link>
+                                            <Link to="/recipes/ingredients" className="dropdown-item">Recipes with my ingredients</Link>
+                                        </>
+                                    }/>
+                                    </li>
+                                </>
```

Comment:
Here you are violating the DOM nesting rules. An "li" should always be nested inside an "ul" or an "ol", never inside another "li". I think you should refactor this.

Also, we are using "useNavigate" throughout our app. For consistency, you should use it here too.